node/bindnode/example_test.go

```diff
@@ -27,9 +27,44 @@ func ExamplePrototypeOnlySchema() {
 	ts.Accumulate(schema.SpawnList("List_String", "String", false))
 
 	schemaType := ts.TypeByName("Person")
-	proto := bindnode.PrototypeOnlySchema(schemaType)
 
-	n, err := qp.BuildMap(proto, -1, func(ma ipld.MapAssembler) {
+	type Person struct {
+		Name    string
+		Age     *int64 // optional
+		Friends []string
+	}
+	person := &Person{
+		Name:    "Michael",
+		Friends: []string{"Sarah", "Alex"},
+	}
+	node := bindnode.Wrap(person, schemaType)
```

Comment:
@willscott this is what I'm thinking in terms of what dealbot could have used.

Reply:
(or, of course, further slimmed down once inferring schemas is implemented)

Reply:
that would have been very nice :)